Code/GraphMol/GeneralizedSubstruct/XQMol.h

```diff
@@ -102,6 +107,10 @@ RDKIT_GENERALIZEDSUBSTRUCT_EXPORT std::vector<MatchVectType> SubstructMatch(
     const ROMol &mol, const ExtendedQueryMol &query,
     const SubstructMatchParameters &params = SubstructMatchParameters());
 
+//! Fingerprints a target molecule, caller owns fingerprint
```

Comment:
Given that this is just a call to `PatternFingerprintMol()`, why do we need it?

Reply:
Looking more closely, I guess it's to ensure that the pattern fingerprint is generated with tautomer insensitivity turned on. This is clearly necessary and easy to get wrong, so it makes sense to add something here.

It probably should also return a unique_ptr though.

Reply:
You're correct - it's to make sure that tautomer fingerprints are used.  Would be nice if this were the default, but I guess that would break existing code.
Updated to `std::unique_ptr`